### DISCUSSION TITLE
Change request: dashboard view should be more high-level and less textual

### DISCUSSION THREAD

**Discussion by dror3go** [babybuddy/babybuddy#530] [Ideas]
Hi and thanks for an AMAZING project!
We've had our first baby just several days ago, and been using it constantly since then. We've both installed the PWA on our phones, and are using pretty much all the features in Baby Buddy.

My biggest issue with it is the dashboard: I find it to be too textual and requires me to read too many small details, some are not really relevant for the main dashboard in my opinion. I find myself looking at the timeline too much, in order to make decisions (should we already wake him to feed, for example).
I think that the dashboard should include some graphs and main numbers that might be important for daily usage: less in the line of "what was the last feeding method" and more in the line of "how many hours he slept today compared to the average of the previous few days", for example - but this requires analysis of what are the key indicators to show in the dashboard.

What do you think? I is something you'll consider?

**Comment by cdubz** [comment]
Certainly open to new directions with the dashboard. It has become dense over the years. For a long time I've had a goal of making it much more configurable in general and this point aligns well with that.

The only issue is that it's a relatively big undertaking. I always work to make time to review PRs and help folks looking contribute. If you're interested in working on this I'll happily assist where I am able but it's tough to commit time otherwise.

Regardless, it might be good to start off by discussing in detail whats on the dashboard and what kind of things would be more useful. I very much agree with your one example but ideally we could work up a more detailed list of ideas and maybe even sketch out what changes might look like.

**Comment by dror3go** [comment]
Thanks @cdubz.
As you probably can imagine - my free time is pretty much gone these days with our 1st born, so taking it is not something I'm able to do these days. That's a tricky project to find contributors for, as most of the users are in my position :rofl: 

Is this the right place for brainstorming with the community?

**Reply by cdubz** [reply]
> Is this the right place for brainstorming with the community?

Yep! This and https://gitter.im/babybuddy/Lobby are the primary places for discussion.
